frontend/src/pages/user-profile/components/user-profile-sidebar/styles.module.scss

```diff
@@ -0,0 +1,46 @@
+.container {
+  width: 350px;
```

Comment:
don't set fixed width, we should make everything responsive

Reply:
done